recipes/smudgeplot/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.2.1" %}
+
+package:
+    name: smudgeplot
+    version: {{ version }}
+
+build:
+    number: 0
+
+source:
+    url: https://github.com/KamilSJaron/smudgeplot/archive/v{{ version }}.tar.gz
+    sha256: 2445564d12a852dcd5ade521b522cabde0df23512687b4ab5b46f73f9b315608
+
+requirements:
+    build:
+        - {{ compiler('c') }}
+        - make
+        - r-devtools
+        - r-base
+        - r-viridis
+        - r-argparse
+    host:
+        - python >=3.6
+        - r-base
+        - r-viridis
+        - r-argparse
+        - numpy
+        - scipy
```

Comment:
Are the python packages actually needed for installation or just during run time?

Reply:
OK, I guess I was surprised to see an R script used for installation rather than `R CMD install`.

Reply:
Indeed. I will transfer this comment to the author, I think they want to improve the install process anyway